modules/navier_stokes/src/fvkernels/WCNSFVMixingLengthEnergyDiffusion.C

```diff
@@ -43,6 +58,9 @@ WCNSFVMixingLengthEnergyDiffusion::WCNSFVMixingLengthEnergyDiffusion(const Input
     _rho(getFunctor<ADReal>(NS::density)),
     _cp(getFunctor<ADReal>(NS::cp)),
     _mixing_len(getFunctor<ADReal>("mixing_length")),
+    _var_interp_method(
+        Moose::FV::selectInterpolationMethod(getParam<MooseEnum>("variable_interp_method"))),
+    _correct_skewness(_var_interp_method == Moose::FV::InterpMethod::SkewCorrectedAverage),
```

Comment:
`_var_interp_method` and `_correct_skewness` definitions and construction are repeated a lot. Something like a `FVDiffusionInterface` (which would inherit from the `FVRelationshipManagerInterface` and add the diffusion RM method) comes to mind, but I'm also not passionate about it